DevOps/README.md

```diff
@@ -0,0 +1,49 @@
+Team members in DevOps_Team:
```

Comment:
Including team members, their responsibilities, and other documentation is a good idea, but most of this file is a repeat of the grading criteria. Recommend removing that part.

Reply:
This applies to the DevOps Team Eric Kniggie is part of the Front-end team